CHANGELOG.md

```diff
@@ -14,12 +14,15 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [4.74]
 ### Added
+- Hovertip to gene panel names with associated genes in variant view, when variant covers more than one gene 
```

Comment:
Great description of the changes! 🥇 

Since we have released a new version of scout yesterday (4.74.1] could you move these lined in a new release item, like this:

```
## [unreleased]
### Added
### Changed
```